go.mod

```diff
@@ -24,9 +24,11 @@ require (
 	github.com/IBM/sarama v1.41.2
 	github.com/antonmedv/expr v1.15.3
 	github.com/arangodb/go-driver v1.6.0
-	github.com/aws/aws-sdk-go-v2 v1.21.0
+	github.com/aws/aws-sdk-go v1.49.10
```

Comment:
We have migrated to `github.com/aws/aws-sdk-go-v2`, please don't use `github.com/aws/aws-sdk-go`